package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shoutem/animation",
-  "version": "0.13.0",
+  "version": "0.13.1-rc.0",
```

Comment:
why RC version here ? this is pulling into master. Can we release animation first ?

Reply:
It's a hotfix branch, hotfix is equivalent to release branch, release branches can have RC versions. Before merging into master this will become 0.13.1, no worries.